advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: 'Using Commvault Backup & Recovery'
+description: 'Walkthroughs of multiple Commvault Backup & Recovery usage scenarios'
+---
+
+How to backups and restore from EDB Postgres Advanced Server using Commvault Backup & Recovery.
+
+## Using Commvault Backup & Recovery
+
+Commvault provides two methods of taking the backup from EDB Postgres Advanced Server and restoring it.
+1. DumpBasedBackupSet Backup and Restore
+2. FSBasedBackupSet Backup and Restore
```

Comment:
add a note
"At this time there is a known issue with FSBased Restore that does not allow for proper restoration of the database. See the known issues section of the guide for more information."